src/transforms/index.ts

```diff
@@ -0,0 +1,4 @@
+export { ITransform, ITransformArgs } from './utils'
+
+export { blacklistKeys } from './blacklist'
+export { whitelistKeys } from './whitelist'
```

Comment:
should reverse these two since whitelist always appears before blacklist everywhere else (order of usage)